### ISSUE TITLE
Where is the code?

### ISSUE THREAD

**Issue by Hades32** [SpeechifyInc/Meta-voicebox#2]
Is this just a placeholder? It's the top match on Google 😅

**Comment by constan1** [comment]
Its a work in progress

**Comment by halr9000** [comment]
Well, then I shall watch this issue for an update. :D

**Comment by zvorinji** [comment]
I imagine this repo was created to nudge Facebook into publishing the code themselves and suspect nobody's working on the implementation but would happily welcome being wrong :-)

**Comment by wei-m-teh** [comment]
Checking in to see if there is any update on sharing the code? Thanks!

**Comment by abhishekc-bhs** [comment]
lol, what a nice way of clickbaiting. 

Archive this repo untill there is an update on the code that we all can try.

**Comment by rsxdalv** [comment]
Yeah given that Facebook doesn't want to release it due to security concerns, I'm a little skeptical that they would OK an external project. But maybe they have 0 say in stopping that.

**Comment by abhishekc-bhs** [comment]
while we are at it, you guys can try this https://github.com/suno-ai/bark.

**Comment by iakashpaul** [comment]
I've unwatched this as well, better track LucidRains progress here  https://github.com/lucidrains/voicebox-pytorch or this upcoming port of Bark as well https://github.com/PABannier/bark.cpp

**Comment by furqan4545** [comment]
clickbait guy... 
Dude be quick, we need code haha
